pycbc/waveform/waveform.py

```diff
@@ -766,6 +766,8 @@ def _base_get_td_waveform_from_fd(template=None, rwrap=None, **params):
                 mass1=nparams['mass1'], mass2=nparams['mass2'],
                 spin1z=nparams['spin1z'], spin2z=nparams['spin2z'])
             rwrap = tau_from_final_mass_spin(m_final, spin_final) * 10
+            if rwrap < 0.2:
```

Comment:
Why not do something like 

rwrap += max(0.2, 0.2 * (mtotal - 60)

That would make sure this is a minimum while providing some safety scaling.

Reply:
Here I use 5 seconds as the minimal instead, so don't assume any rescaling relation for BBHs in ground-based detectors, 5s is long enough for XG, and still not a computational burden for the current 2G study:
![image](https://github.com/gwastro/pycbc/assets/6574526/4cffe469-7fda-4454-916e-3f790ed2eeca)

Reply:
@ahnitz Poke :)